step-api/src/main/java/org/jenkinsci/plugins/workflow/structs/DescribableHelper.java

```diff
@@ -73,6 +73,9 @@
 /**
  * Utility for converting between {@link Describable}s (and some other objects) and map-like representations.
  * Ultimately should live in Jenkins core (or Stapler).
+ *
+ * @deprecated
+ *      Moved to the structs plugin
```

Comment:
It seems it is not really moved but moved-renamed (it is now `DescribableModel `). Maybe a link to the new class in the javadoc makes sense (and developers will be thankful!)